### ISSUE TITLE
bug: windows 125% scaleing breaks vue-masonry-wall 

### ISSUE THREAD

**Issue by nanu-c** [DerYeger/yeger#198]
### Affected Packages

- [ ] @yeger/eslint-config
- [X] @yeger/vue-masonry-wall
- [ ] @yeger/vue2-masonry-wall
- [ ] vecti

### Description

This is really difficult to reproduce but i still wanted to report it.

1. Need windows and needs scaling set to 125%. It happens on a screen that has 1980px width  that is downscaled to 1536px
2. the space for the 
3. The content of the masonry must be loaded asynchronly meaning the rendering changes

What happens:
The wall always switches from 4 to 3 columns, resulting into reloading the  async content. Hence something in the calculation of the width doesn't respect the screen pixel ratio. In our case we have a reload loop of the elements

### Reproduction

We don't have a simple example

### Preferences

- [ ] I want to be assigned to and work on this issue myself

**Comment by DerYeger** [comment]
Sorry, without a reproduction I can't help much.

Based on your description, here's what I assume is happening:

- The items being rendered causes a vertical scroll bar to appear.
- This reduces the available width, causing the number of columns to be reduced to 3.
- Your components rendes with decreased height, due to their content loading asynchronously.
- The scrollbar disappears, and the cycle starts again.

As described in https://github.com/DerYeger/yeger/tree/main/packages/vue-masonry-wall#limitations, components with dynamic height are intentionally ignored.
This has nothing to do with the window width or the scaling factor. It just so happens that this exact ratio in combination with your chosen column width results in the appearance of the scrollbar causing the number of columns to be decreased.

In short, dynamically changing item height is invalid, this is not a bug.

**Comment by DerYeger** [comment]
I'm closing this for now, since it's intended and required behavior.
